src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -59,23 +59,22 @@ const AvatarGrid = () => {
 
   // FIXME: This is really bad, needs to be fixed ASAP
   const partygoersBySeat: WithId<User>[][] = [];
-  partygoers &&
-    partygoers.forEach((partygoer) => {
```

Comment:
This could probably be replaced by `usePartygoersBySeat`:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/Camp/hooks/usePartygoersBySeat.ts

Reply:
I think it is `AvatarGrid` is a legacy template as per 
![image](https://user-images.githubusercontent.com/63194656/105354674-903f8b80-5bf9-11eb-8c9b-b4370354e7b1.png)